bsp/boards/openmote-b/radio.c

```diff
@@ -1,531 +1,325 @@
 /**
- * Author: Xavier Vilajosana (xvilajosana@eecs.berkeley.edu)
- *         Pere Tuset (peretuset@openmote.com)
- * Date:   July 2013
- * Description: CC2538-specific definition of the "radio" bsp module.
- */
-
-#include <stdio.h>
-#include <string.h>
-
-#include <headers/hw_ana_regs.h>
-#include <headers/hw_ints.h>
-#include <headers/hw_rfcore_sfr.h>
-#include <headers/hw_rfcore_sfr.h>
-#include <headers/hw_rfcore_xreg.h>
-#include <headers/hw_types.h>
-
-#include <source/interrupt.h>
-#include <source/sys_ctrl.h>
-
+    \brief Definition of the Open Radio interface for openmote-b
```

Comment:
You cannot remove authors, add you name at the bottom of the list.

Reply:
Oh sorry this was a mistake